src/components/GetStarted/EditPreferences.vue

```diff
@@ -6,138 +6,6 @@
 		<p class="tw-text-subhead tw-mb-4">
 			Change your preferences and find more borrowers to support.
 		</p>
-		<div class="row">
```

Comment:
It actually looks like this component can be removed entirely? Definitely look into what was being used by the route that is being removed, and remove any components from there that can be removed that aren't used elsewhere.